geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -40,10 +42,16 @@ trait IngestCommand[DS <: DataStore] extends DataStoreCommand[DS] with Interacti
     import DataFormats._
     import org.locationtech.geomesa.utils.conversions.ScalaImplicits.RichIterator
 
-    ensureSameFs(PathUtils.RemotePrefixes)
+    val ingestFiles: jList[String] = if (params.srcList) {
+      params.files.flatMap(Source.fromFile(_).getLines().toList)
+    } else {
+      params.files
+    }
+
+    ensureSameFs(ingestFiles, PathUtils.RemotePrefixes)
 
     if (params.fmt == Shp) {
-      createShpIngest().run()
+      createShpIngest(ingestFiles).run()
```

Comment:
hmm, something seems screwed up with this PR - this line doesn't exist in master any more. can you try creating a new branch from master and merge squashing it?

Reply:
oh i guess there are merge conflicts that need to be fixed...